src/main/java/frc/robot/elevator/ElevatorSubsystem.java

```diff
@@ -121,6 +154,12 @@ protected void afterTransition(ElevatorState newState) {
     }
   }
 
+  public void setSyncCoeffecient(double armRotations) {
```

Comment:
admittedly this is a pretty clever way to measure desync between the elevator and arm. seems reasonable to keep this code around since it'll likely remain useful for the actual trapezoid profile slow down math